docs/source/contributing.rst

```diff
@@ -15,16 +15,17 @@ Before Contributing
 
 Before sending us a Pull Request, please consider this:
 
-* Is the contribution entirely your own work, or is it already licensed under an LGPL 2.1 compatible Open Source License? If not, we unfortunately cannot accept it.
+* Is the contribution entirely your own work, or is it already licensed under an LGPL 2.1 compatible Open Source License? If not, we, unfortunately, cannot accept it.
```

Comment:
This sentence is a bit awkward.

Suggestion:
If it's not, we unfortunately cannot accept it.